{"uid":"40681465aed3ee19","name":"Verify POST /store/order rejects invalid order data with appropriate error codes","fullName":"com.petstore.tests.store.StoreOrderTest.testPlaceOrderWithInvalidData","historyId":"e67ab1b62402e583869f34e173f038ac","time":{"start":1749415756349,"stop":1749415756378,"duration":29},"description":"Validates that API properly rejects invalid order data and returns appropriate HTTP error codes (4xx)","descriptionHtml":"<p>Validates that API properly rejects invalid order data and returns appropriate HTTP error codes (4xx)</p>\n","status":"failed","statusMessage":"Critical validation gap detected: API accepted invalid order data for 'Negative quantity'. Expected 4xx status but got 200. This indicates missing API validation. Response: {\"id\":9223372036854748424,\"petId\":1,\"quantity\":-5,\"complete\":false}","statusTrace":"java.lang.AssertionError: Critical validation gap detected: API accepted invalid order data for 'Negative quantity'. Expected 4xx status but got 200. This indicates missing API validation. Response: {\"id\":9223372036854748424,\"petId\":1,\"quantity\":-5,\"complete\":false}\n\tat com.petstore.tests.BaseStoreTest.handleValidationFailure(BaseStoreTest.java:441)\n\tat com.petstore.tests.BaseStoreTest.validateInvalidDataRejection(BaseStoreTest.java:420)\n\tat com.petstore.tests.store.StoreOrderTest.testPlaceOrderWithInvalidData(StoreOrderTest.java:249)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\n\tat org.testng.TestRunner.run(TestRunner.java:616)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)\n\tat org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)\n\tat org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)\n\tat org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)\nCaused by: java.lang.AssertionError: API should reject invalid order data with 4xx status for test case: Negative quantity, but got: 200 expected [true] but found [false]\n\tat org.testng.Assert.fail(Assert.java:111)\n\tat org.testng.Assert.failNotEquals(Assert.java:1590)\n\tat org.testng.Assert.assertTrue(Assert.java:57)\n\tat com.petstore.tests.BaseStoreTest.validateInvalidDataRejection(BaseStoreTest.java:399)\n\t... 33 more\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"setupClass","time":{"start":1749415749205,"stop":1749415749834,"duration":629},"description":"","status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"baseSetUp","time":{"start":1749415753546,"stop":1749415753548,"duration":2},"description":"","status":"passed","steps":[{"name":"Environment: {0}, Base URL: {1}","time":{"start":1749415753548,"stop":1749415753548,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":1,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true},{"name":"beforeMethod","time":{"start":1749415756347,"stop":1749415756347,"duration":0},"description":"","status":"passed","steps":[{"name":"Starting test: public void com.petstore.tests.store.StoreOrderTest.testPlaceOrderWithInvalidData(com.petstore.dto.Order,java.lang.String)","time":{"start":1749415756347,"stop":1749415756347,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"method","value":"public void com.petstore.tests.store.StoreOrderTest.testPlaceOrderWithInvalidData(com.petstore.dto.Order,java.lang.String)"}],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true}],"attachments":[],"parameters":[],"stepsCount":1,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true},{"name":"setupTest","time":{"start":1749415756348,"stop":1749415756348,"duration":0},"description":"","status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"description":"Validates that API properly rejects invalid order data and returns appropriate HTTP error codes (4xx)","status":"failed","statusMessage":"Critical validation gap detected: API accepted invalid order data for 'Negative quantity'. Expected 4xx status but got 200. This indicates missing API validation. Response: {\"id\":9223372036854748424,\"petId\":1,\"quantity\":-5,\"complete\":false}","statusTrace":"java.lang.AssertionError: Critical validation gap detected: API accepted invalid order data for 'Negative quantity'. Expected 4xx status but got 200. This indicates missing API validation. Response: {\"id\":9223372036854748424,\"petId\":1,\"quantity\":-5,\"complete\":false}\n\tat com.petstore.tests.BaseStoreTest.handleValidationFailure(BaseStoreTest.java:441)\n\tat com.petstore.tests.BaseStoreTest.validateInvalidDataRejection(BaseStoreTest.java:420)\n\tat com.petstore.tests.store.StoreOrderTest.testPlaceOrderWithInvalidData(StoreOrderTest.java:249)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\n\tat org.testng.TestRunner.run(TestRunner.java:616)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)\n\tat org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)\n\tat org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)\n\tat org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)\nCaused by: java.lang.AssertionError: API should reject invalid order data with 4xx status for test case: Negative quantity, but got: 200 expected [true] but found [false]\n\tat org.testng.Assert.fail(Assert.java:111)\n\tat org.testng.Assert.failNotEquals(Assert.java:1590)\n\tat org.testng.Assert.assertTrue(Assert.java:57)\n\tat com.petstore.tests.BaseStoreTest.validateInvalidDataRejection(BaseStoreTest.java:399)\n\t... 33 more\n","steps":[{"name":"Execute order placement with invalid data: Negative quantity","time":{"start":1749415756350,"stop":1749415756377,"duration":27},"status":"passed","steps":[],"attachments":[{"uid":"427f63d5fb33fbb9","name":"Request","source":"427f63d5fb33fbb9.html","type":"text/html","size":812},{"uid":"cd032572f5b78bd9","name":"HTTP/1.1 200 OK","source":"cd032572f5b78bd9.html","type":"text/html","size":719}],"parameters":[{"name":"invalidOrder","value":"Order(id=null, petId=1, quantity=-5, shipDate=null, status=null, complete=null)"},{"name":"testCase","value":"Negative quantity"}],"stepsCount":0,"attachmentsCount":2,"shouldDisplayMessage":false,"hasContent":true},{"name":"Validate invalid data rejection for test case: Negative quantity","time":{"start":1749415756377,"stop":1749415756378,"duration":1},"status":"failed","statusMessage":"Critical validation gap detected: API accepted invalid order data for 'Negative quantity'. Expected 4xx status but got 200. This indicates missing API validation. Response: {\"id\":9223372036854748424,\"petId\":1,\"quantity\":-5,\"complete\":false}","statusTrace":"java.lang.AssertionError: Critical validation gap detected: API accepted invalid order data for 'Negative quantity'. Expected 4xx status but got 200. This indicates missing API validation. Response: {\"id\":9223372036854748424,\"petId\":1,\"quantity\":-5,\"complete\":false}\n\tat com.petstore.tests.BaseStoreTest.handleValidationFailure(BaseStoreTest.java:441)\n\tat com.petstore.tests.BaseStoreTest.validateInvalidDataRejection(BaseStoreTest.java:420)\n\tat com.petstore.tests.store.StoreOrderTest.testPlaceOrderWithInvalidData(StoreOrderTest.java:249)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\n\tat org.testng.TestRunner.run(TestRunner.java:616)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)\n\tat org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)\n\tat org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)\n\tat org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)\nCaused by: java.lang.AssertionError: API should reject invalid order data with 4xx status for test case: Negative quantity, but got: 200 expected [true] but found [false]\n\tat org.testng.Assert.fail(Assert.java:111)\n\tat org.testng.Assert.failNotEquals(Assert.java:1590)\n\tat org.testng.Assert.assertTrue(Assert.java:57)\n\tat com.petstore.tests.BaseStoreTest.validateInvalidDataRejection(BaseStoreTest.java:399)\n\t... 33 more\n","steps":[{"name":"CRITICAL: Validation failure detected for test case: Negative quantity","time":{"start":1749415756377,"stop":1749415756378,"duration":1},"status":"failed","statusMessage":"Critical validation gap detected: API accepted invalid order data for 'Negative quantity'. Expected 4xx status but got 200. This indicates missing API validation. Response: {\"id\":9223372036854748424,\"petId\":1,\"quantity\":-5,\"complete\":false}","statusTrace":"java.lang.AssertionError: Critical validation gap detected: API accepted invalid order data for 'Negative quantity'. Expected 4xx status but got 200. This indicates missing API validation. Response: {\"id\":9223372036854748424,\"petId\":1,\"quantity\":-5,\"complete\":false}\n\tat com.petstore.tests.BaseStoreTest.handleValidationFailure(BaseStoreTest.java:441)\n\tat com.petstore.tests.BaseStoreTest.validateInvalidDataRejection(BaseStoreTest.java:420)\n\tat com.petstore.tests.store.StoreOrderTest.testPlaceOrderWithInvalidData(StoreOrderTest.java:249)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\n\tat org.testng.TestRunner.run(TestRunner.java:616)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)\n\tat org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)\n\tat org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)\n\tat org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)\nCaused by: java.lang.AssertionError: API should reject invalid order data with 4xx status for test case: Negative quantity, but got: 200 expected [true] but found [false]\n\tat org.testng.Assert.fail(Assert.java:111)\n\tat org.testng.Assert.failNotEquals(Assert.java:1590)\n\tat org.testng.Assert.assertTrue(Assert.java:57)\n\tat com.petstore.tests.BaseStoreTest.validateInvalidDataRejection(BaseStoreTest.java:399)\n\t... 33 more\n","steps":[{"name":"Track order for cleanup","time":{"start":1749415756377,"stop":1749415756377,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"response","value":"io.restassured.internal.RestAssuredResponseImpl@46309fa8"}],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true}],"attachments":[],"parameters":[{"name":"response","value":"io.restassured.internal.RestAssuredResponseImpl@46309fa8"},{"name":"testCase","value":"Negative quantity"},{"name":"actualStatusCode","value":"200"},{"name":"e","value":"java.lang.AssertionError: API should reject invalid order data with 4xx status for test case: Negative quantity, but got: 200 expected [true] but found [false]"}],"stepsCount":1,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"attachments":[],"parameters":[{"name":"response","value":"io.restassured.internal.RestAssuredResponseImpl@46309fa8"},{"name":"testCase","value":"Negative quantity"}],"stepsCount":2,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true}],"attachments":[],"parameters":[],"stepsCount":4,"attachmentsCount":2,"shouldDisplayMessage":false,"hasContent":true},"afterStages":[{"name":"cleanUp","time":{"start":1749415756379,"stop":1749415756379,"duration":0},"description":"","status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"cleanupCreatedOrders","time":{"start":1749415756380,"stop":1749415756407,"duration":27},"description":"","status":"passed","steps":[],"attachments":[{"uid":"280de324d68556f4","name":"Request","source":"280de324d68556f4.html","type":"text/html","size":413},{"uid":"7d90b261e7d262a5","name":"HTTP/1.1 404 Not Found","source":"7d90b261e7d262a5.html","type":"text/html","size":713}],"parameters":[],"stepsCount":0,"attachmentsCount":2,"shouldDisplayMessage":false,"hasContent":true}],"labels":[{"name":"package","value":"com.petstore.tests.store.StoreOrderTest"},{"name":"testClass","value":"com.petstore.tests.store.StoreOrderTest"},{"name":"testMethod","value":"testPlaceOrderWithInvalidData"},{"name":"parentSuite","value":"Petstore /store API Test Suite"},{"name":"suite","value":"Regression Tests"},{"name":"subSuite","value":"com.petstore.tests.store.StoreOrderTest"},{"name":"host","value":"pkrvmf6wy0o8zjz"},{"name":"thread","value":"2175@pkrvmf6wy0o8zjz.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Place Order"},{"name":"epic","value":"Store API"},{"name":"feature","value":"Order Management"},{"name":"severity","value":"critical"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"arg0","value":"Order(id=null, petId=1, quantity=-5, shipDate=null, status=null, complete=null)"},{"name":"arg1","value":"Negative quantity"},{"name":"orderId","value":"9223372036854748424"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"40681465aed3ee19.json","parameterValues":["Order(id=null, petId=1, quantity=-5, shipDate=null, status=null, complete=null)","Negative quantity","9223372036854748424"]}